src/app/features/activity-list/components/transaction-list/stacks-transaction/ft-transfer-item.tsx

```diff
@@ -31,6 +32,11 @@ export function FtTransferItem({ ftTransfer, parentTx }: FtTransferItemProps) {
   const currentAccount = useCurrentStacksAccount();
   const isOriginator = ftTransfer.sender === currentAccount?.address;
 
+  if (assetMetadata && 'error' in assetMetadata) {
+    logger.error('Token metadata not found');
+    return null;
+  }
```

Comment:
Would be better not to repeat `'error' in assetMetadata`. Consider these helpers:

```ts
type FtAssetResponse = FtMetadataResponse | TokenNotFoundResponse;

function isTokenNotFoundResponse(resp: FtAssetResponse): resp is TokenNotFoundResponse {
  return 'error' in resp;
}

function isFtAsset(resp: FtAssetResponse): resp is FtMetadataResponse {
  return !isTokenNotFoundResponse(resp);
}
```

Reply:
Good idea, thanks. I'll implement this.